CAAppKitBridge/Demo/main.m

```diff
@@ -34,15 +34,15 @@ @interface DemoController : NSObject
 @end
 
 @implementation DemoController
-- (void) applicationDidFinishLaunching: (id) t
+- (void) applicationDidFinishLaunching: (id)t
 {
-  NSView * view = [[NSView alloc]init];
-  NSView * view2 = [[NSView alloc]init];
-  NSView * view3 = [[NSView alloc]init];
-  NSView * view3_5 = [[NSView alloc]init];
-  NSView * view4 = [[NSView alloc]init];
+  NSView * view = [[NSView alloc] init];
+  NSView * view2 = [[NSView alloc] init];
+  NSView * view3 = [[NSView alloc] init];
+  NSView * view3_5 = [[NSView alloc] init];
+  NSView * view4 = [[NSView alloc] init];
 
-  NSLog(@"%p %p %p %p %p",view,view2,view3,view3_5,view4);
+  NSLog(@"%p %p %p %p %p", view, view2, view3, view3_5, view4);
```

Comment:
Seems like noise you might not need.

Reply:
Im actually going to leave this. I've had issues where these objects werent initialised properly, giving me a bunch of nilpointers.